PocketKit/Sources/PocketKit/Collections/CollectionStory+Extensions.swift

```diff
@@ -0,0 +1,11 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Sync
+
+extension CollectionStory {
+    var isSaved: Bool {
+        item?.savedItem != nil && item?.savedItem?.isArchived == false
```

Comment:
nit: style preference for readability
```suggestion
       guard let savedItem = item?.savedItem else { return nil }
       return !savedItem.isArchived
```